.gitignore

```diff
@@ -113,4 +113,7 @@ ENV/
 .mypy_cache/
 
 # CI
-.ci
\ No newline at end of file
+.ci
+
+#vscode
```

Comment:
```suggestion
```

(kinda obvious)

Reply:
Sure, but I just followed the conventions I have seen for this file

Reply:
Some of the comments should be removed from the original file, that's true. It was auto-added in GitHub UI, and nobody looked deep inside. So:

- `#MacOS` -> `# macOS`
- `#PyCharm IDE` -> `# PyCharm IDE`
- `* stuff:` -> `*`
- Delete `# Scrapy stuff:`
- Delete `# dotenv`
- Delete `# mypy`
- Delete `# CI`